src/allotropy/parsers/beckman_vi_cell_xr/vi_cell_xr_reader.py

```diff
@@ -102,9 +106,15 @@ def _get_file_info(self) -> SeriesData:
         return SeriesData(info)
 
     def _get_file_version(self) -> XrVersion:
-        match = re.match(MODEL_RE, self.file_info[str, "model"], flags=re.IGNORECASE)
+        raw_model = self.file_info.get(str, "model")
+        if not raw_model:
```

Comment:
Note: this is a "fix" (really just getting more info) for a bug I saw where `model` key was missing. I don't see how that could happen, because we set model in the index elsewhere. But, this new error will give the info if it happens agian.